cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -213,9 +214,11 @@ func (r *Runner) mountFileEnv(envName string, containerPath string) error {
 	fi, err := os.Stat(filePath)
 	if err != nil {
 		if os.IsNotExist(err) {
-			return fmt.Errorf("env %s is set, but file %s does not exist", envName, filePath)
+			// Ignore if file does not exists
```

Comment:
The wrapper binary is common for all the infra providers. for some providers the creds files are optional while some requires it (`gcp`). This breaks our CI where we always set all the env variables for all the builds. So adding fix to ignore if file does not exists. The infrastructure specific validation is moved to its subcommand. check `konvoy-image/cmd/gcp.go`

Reply:
That makes sense, and because [in this line](https://github.com/mesosphere/konvoy-image-builder/pull/554/files/aa5ea098dadeccdceba37e5c5611f64faabc7701#diff-d8016cb551cfda4bbd3bc9124263f9003367a4cf5a6d144b96c928faec12f3aaR204) the same file path will be mounted its ok to move it into the `cmd` package.

Please just a comment in `setGCPEnv` that we expect the mount path to equal the path on the host.

Reply:
thank you. sounds good.